go.mod

```diff
@@ -7,6 +7,7 @@ module github.com/trustbloc/edv
 go 1.13
 
 require (
+	github.com/btcsuite/btcutil v1.0.1
```

Comment:
Provides the base58 library.

Reply:
use `"github.com/btcsuite/btcutil/base58"` instead
unless you need to use this specific version.

Reply:
never mind... I thought this was an import statement

this is the go.mod file :)

Reply:
OK! :)